docs/docs/policy-language.md

```diff
@@ -1361,9 +1361,9 @@ expressions are simultaneously satisfied.
 Therefore, there are other ways to express the desired policy.
 
 For this policy, you can also define a rule that finds if there exists a bitcoin-mining
-app (which is easy using the `some` keyword). And then you use negation to check
-that there is NO bitcoin-mining app. Technically, you're using 2 negations and
-an existential quantifier, which is logically the same as a universal
+app (which is easy using the [`some` keyword](#some-keyword)). And then you use negation to check
+that there is NO bitcoin-mining app. Technically, you're using 2 [negations](#negation) and
```

Comment:
Also not your change (the links are good), so sorry for the tangentially related comments... but are we really using two negations in that example? I only see one 😅